ext/css/action-popup.css

```diff
@@ -496,4 +496,8 @@ select.profile-select {
         width: 100%;
         padding-top: 37.7%;
     }
+    #extension-info {
+        max-width: 95vw;
```

Comment:
vw?

Reply:
Oh not a typo https://developer.mozilla.org/en-US/docs/Web/CSS/max-width

Reply:
Due to the text extending the page itself it isnt possible to use percents here so that's why it's vw.